### DISCUSSION TITLE
Fine tuning tutorial?

### DISCUSSION THREAD

**Discussion by ignacio82** [fwartner/home-assistant-wakewords-collection#38] [Q&A]
Is there a tutorial out there for fine tuning wake words? In particular I would like to further train the model with to work better for with the members of my household. Similar at how google asks you to repeat ok google and hey google a few times when you setup assistant for the first time.

**Comment by fwartner** [comment]
As far as i know: Not yet.
But i bet that there will be something coming in the future as the stuff for voice-assistants in HA is growing.
